test/models/node_test.rb

```diff
@@ -362,4 +362,86 @@ def test_containing_relations
     assert_equal relation_member2.relation.id, cr.second.id
     assert_equal relation_member3.relation.id, cr.third.id
   end
+
+  test "raises missing changeset exception when creating" do
```

Comment:
We should really be consistent and stick to using `do test_` or else change all the existing test definitions but I see some using this style have already snuck in without being noticed so I guess a few more won't hurt :-(